composer.json

```diff
@@ -24,8 +24,8 @@
         "ext-soap": "*",
         "ext-curl": "*",
         "ass/xmlsecurity": "~1.0",
-        "symfony/framework-bundle": "~2.6",
-        "symfony/twig-bundle": "~2.6",
+        "symfony/framework-bundle": ">=2.8",
```

Comment:
Use `^2.7 || ^3.0`. Otherwise you would support Symfony 4 too. 2.7 can still be supported, because it's a LTS too.